### PR TITLE
Consume Sync XCFramework via Cocoapods

### PR DESCRIPTION
**Pull request by leemaguire**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ x.y.z Release notes (yyyy-MM-dd)
 =============================================================
 ### Enhancements
 * Throw an exception for Objects that have none of its properties marked with @objc.
+* Mac Catalyst is now supported when integrating with Cocoapods.
```
Comment (tgoyne) [inline CHANGELOG.md:5]: and arm64 simulators

Also need to mention that the minimum version of cocoapods supported is now 1.10

**Comment by tgoyne** [comment]
Also needs a changelog entry for the objectstore bug fix this pulled in.

**Comment by tgoyne** [comment]
Retest this please

**Comment by fealebenpae** [comment]
Retest this please

**Comment by fealebenpae** [comment]
Retest this please

**Comment by leemaguire** [comment]
@tgoyne I've tested the branch again and it works fine.

**Review by tgoyne — APPROVED** [review]
Remaining CI failures appear to just be jenkins networking issues.

**Pull request merged**